src/ethereum/utils/address.py

```diff
@@ -0,0 +1,40 @@
+"""
+Utility Functions For Addresses
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+..contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Address specific utility functions used in this application.
+"""
+from ethereum.base_types import Uint
+from ethereum.crypto import keccak256
+from ethereum.frontier import rlp
+from ethereum.frontier.eth_types import Address
+
+
+def compute_contract_address(address: Address, nonce: Uint) -> Address:
```

Comment:
This might be taking futureproofing too far, but with all the recent talk of Address Space Extension, this function might be better located in the `frontier` package?